### DISCUSSION TITLE
Can we use autogluon to achieve the transfer learning of the Tabular?

### DISCUSSION THREAD

**Discussion by HazelSCUT** [autogluon/autogluon#2552] [Q&A] [answered]
I used the TabularPredictor to train the model of summer data, then I want to transfer such a model to predict the winter data (i.e. transfer learning). Owning to the different correlations between inputs and the target, the predictive results seem to be worse when I directly load the trained model of summer to predict the winter data. So  I consider transfer learning. I have reviewed all codes, but I could not find APIs to get the transfer learning of the Tabular. Could you please give me some advice? Thanks!

**Comment by Innixma** [comment, marked as answer]
Transfer learning is not yet supported in Tabular. This is something we are currently investigating, but I would suggest simply training with the winter data to predict future winter data.
